deployer/src/config_creator.py

```diff
@@ -68,6 +68,37 @@ def to_docusaurus_config(config, urls=None):
     return config
 
 
+def to_docusaurus_v2_config(config, urls=None):
+    if urls:
+        config["sitemap_urls"] = [
+            extract_root_from_input(urls[0]) + "sitemap.xml"]
+        config["sitemap_alternate_links"] = True
+    start_url = urls[0]
```

Comment:
if urls is None or empty, this statement will break

Reply:
Yes sure. Since this function is only call one time with a non empty list, I thought it would be fine this way (internal use).
I will check it thx